src/templates/input.html

```diff
@@ -2,7 +2,7 @@
 {% from "_includes/forms" import time %}
 <tr>
     <th>
-        {{ craft.i18n.getLocaleData().getWeekDayName(day) }}
+        {{ craft.i18n.getLocaleData().getWeekDayName(day) | ucfirst }}
```

Comment:
I guess officially in Ditch they're not capitalised but I thought it looked nicer, didn't mean to add this to the PR!